src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetDestruction.java

```diff
@@ -383,33 +377,37 @@ public void clearArea(World world, BlockPos pos, EnumFacing side, EntityPlayer p
     public static void storeUndo(World world, Map<BlockPos, IBlockState> posStateMap, Map<BlockPos, IBlockState> pasteStateMap, BlockPos startBlock, ItemStack stack, EntityPlayer player) {
         WorldSave worldSave = WorldSave.getWorldSaveDestruction(world);
         NBTTagCompound tagCompound = new NBTTagCompound();
-        List<Integer> posIntArrayList = new ArrayList<Integer>();
-        List<Integer> stateIntArrayList = new ArrayList<Integer>();
-        List<Integer> pastePosArrayList = new ArrayList<Integer>();
-        List<Integer> pasteStateArrayList = new ArrayList<Integer>();
+        IntList posIntArrayList = new IntArrayList();
```

Comment:
Love the code cleaning here but please do a separate PR in the future. I know it's ugly, I know it's hard to read but it's truly just as annoying to have to review non-changes in a changes pr.

Reply:
I don’t want revert these changes tough... it is quite painful to do it — I already forgot what are the fixes and what are the cleanups.

Reply:
As I said. In the future. I'll ignore it this time :P